Automattic/wp-calypso
# Multisite Scheduled Updates: Add global navigation Plugins menu item

Closes https://github.com/Automattic/wp-calypso/issues/90577

## Proposed Changes

* Creates plugins section specific global navigation menu items

## Testing Instructions

* Go to `/sites`
* Check if there is a "Plugins" menu item
* Click on it and check if there is a "Scheduled updates" feature renders
<img width="611" alt="Screenshot 2024-05-16 at 18 23 32" src="https://github.com/Automattic/wp-calypso/assets/1241413/4120a979-f3e2-46da-ab78-fa38f0d80057">

![Screen Capture on 2024-05-16 at 11-38-35](https://github.com/Automattic/wp-calypso/assets/1241413/b3d33e12-75a2-4578-944d-c5b0284a01c6)

## Pre-merge Checklist
